src/amqp.c

```diff
@@ -102,13 +102,18 @@ static char *amqp_bytes_to_new_string(amqp_bytes_t data)
 	return str;
 }
 
+/**
```

Comment:
Related to commit message 69b2b1a7c8f056a214d21424e8298ad8d9b4cb06:
Remove `user`. There is no user interaction on this flow. Use publisher/consumer or client.